src/components/radioGroup/radioGroup.stories.tsx

```diff
@@ -0,0 +1,62 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import { useState } from 'react';
+import { Radio } from './radio';
+import { RadioGroup, type IRadioGroupProps } from './radioGroup';
+
+const meta: Meta<typeof RadioGroup> = {
+    title: 'components/RadioGroup',
```

Comment:
We don't usually have a folder which also contain stories (see `alerts` and `avatars`). I'd structure this as follows:
```suggestion
    title: 'components/RadioGroup/RadioGroup',
```